backend/src/libs/packages/config/libs/interfaces/config.interface.ts

```diff
@@ -1,7 +1,7 @@
 import { type IConfig as ILibraryConfig } from 'shared/build/index.js';
 
-import { type EnvironmentSchema } from '../types/types.js';
+import { type AuthConfig, type EnvironmentSchema } from '../types/types.js';
 
-interface IConfig extends ILibraryConfig<EnvironmentSchema> {}
+interface IConfig extends ILibraryConfig<EnvironmentSchema, AuthConfig> {}
```

Comment:
```suggestion
interface IConfig extends ILibraryConfig<EnvironmentSchema> {
  AUTH: AuthConfig
}
```
you need it only of the backed